### PR TITLE
chore: Add to e2e

### PR DESCRIPTION
**Pull request by nahsi**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        run: ./build.sh
+
+      - name: Run cargo test
+        run: cargo test --release --all-features
+
+      - name: Run cargo fmt
+        run: cargo fmt --all -- --check
+
+      # - name: Run cargo clippy
+      #   run: cargo clippy --all --all-features
```
Comment (mikevoronov) [inline .github/workflows/tests.yml:49]: let's remove it?

**Review by mikevoronov — APPROVED** [review]
utACK

File: Cargo.toml [hunk 2]
```diff
@@ -37,7 +37,7 @@ path = "src/test.rs"
 
 [dependencies]
 marine-rs-sdk = "0.7.0"
-bytesize = "1.2.0"
```
[the remark this answers is not in the document]
Comment (nahsi) [reply Cargo.toml:40]: Either this or need to bump bytesize in marine repos, then release new version and bump marine in spells.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (nahsi) [reply .github/workflows/tests.yml:49]: I was hopping someone would fix this :)

**Review by mikevoronov — APPROVED** [review]
utACK

**Pull request merged**